src/pages/LogDrop/FileDropper.tsx

```diff
@@ -59,59 +59,59 @@ const FileDropper: React.FC<FileDropperProps> = ({ onChangeLogType }) => {
     }
   }, [ingestLines, logType, sendEvent, onChangeLogType]);
 
-  const { getRootProps, getInputProps, isDragActive, open } = useDropzone({
+  const { getRootProps, getInputProps, open } = useDropzone({
     onDrop,
     maxFiles: 1,
+    noClick: true,
+    noKeyboard: true,
   });
 
   return (
     <Container>
       <BorderBox>
         {hasDroppedLog ? (
```

Comment:
Just a UX note, and no worries if the scope of this is a little difficult and you'd rather not, but: it would be nice if the behavior were a little more defined if a user tries to drag and drop an additional log. I would probably expect either a clear indicator that the operation failed, or for the second log to replace the first. Right now nothing in particular happens.

Reply:
I updated the behavior! Lmk what you think!